rllib/algorithms/ppo/tests/test_ppo.py

```diff
@@ -331,7 +359,65 @@ def get_value(fw=fw, policy=policy, log_std_var=log_std_var):
             # Check the variable is initially zero.
             init_std = get_value()
             assert init_std == 0.0, init_std
-            batch = compute_gae_for_sample_batch(policy, FAKE_BATCH.copy())
+            batch = compute_gae_for_sample_batch(policy, CARTPOLE_FAKE_BATCH.copy())
```

Comment:
I actually think that this does not test log_std by the way.
While writing the test, I discovered that we are using cartpole here which has a discrete action space.
It therefore never has a gaussian action distribution and can not have the free floating std - the setting should simply be ignored but not tested here.
I've written the test below with pendulum.
@kouroshHakha Can you think of a reason why I could be wrong here?
Otherwise I'm changing test_ppo_free_log_std to also test with pendulum.

Reply:
sgtm